src/Symfony/Component/Mime/Message.php

```diff
@@ -130,11 +130,11 @@ public function toIterable(): iterable
      */
     public function ensureValidity()
     {
-        if (!$this->headers->has('To') && !$this->headers->has('Cc') && !$this->headers->has('Bcc')) {
+        if (empty($this->headers->get('To')?->getBody()) && empty($this->headers->get('Cc')?->getBody()) && empty($this->headers->get('Bcc')?->getBody())) {
```

Comment:
We don’t allow the usage of empty in the code.

Reply:
@fabpot Now using `!` instead of `empty` which is indeed sufficient in this case.
Is there somewhere I can find which functions are disallowed (I couldn't find it in the Conventions or Coding Standards documentation)?